src/views/Home.jsx

```diff
@@ -34,14 +34,17 @@ export function Home({ setListToken }) {
 
 	return (
 		<div className="Home">
+			<h2>Welcome to your Smart Shopping List! </h2>{' '}
```

Comment:
I've never seen the `{' '}` at the end of a line here in JSX. What does this line look like without it? And could this be a spacing issue better controlled with CSS?

Reply:
Great question, Amy. Beth and I were wondering the same, as these kept appearing this week. I'm going to do a little research and see what I can find out about this.

Reply:
It adds whitespace, and I don't know, but it's possible the linter adds this when you save. I notice when I use Prettier this can happen.

Reply:
Agree with what @danathedev. Sometimes if I copy/paste or move things around in the code, that whitespace gets add unintentionally. Probably safe to remove!